src/CMakeLists.txt

```
@@ -1,19 +1,19 @@
 ##################################################
 # Build a custom protoc plugin
-ign_add_executable(ign_msgs_gen Generator.cc generator_main.cc)
-target_link_libraries(ign_msgs_gen
+ign_add_executable(gz_msgs_gen Generator.cc generator_main.cc)
+target_link_libraries(gz_msgs_gen
   protobuf::libprotoc
   protobuf::libprotobuf)
-target_include_directories(ign_msgs_gen PRIVATE ${PROTOBUF_INCLUDE_DIR})
-target_compile_features(ign_msgs_gen PRIVATE ${IGN_CXX_11_FEATURES})
+target_include_directories(gz_msgs_gen PRIVATE ${PROTOBUF_INCLUDE_DIR})
+target_compile_features(gz_msgs_gen PRIVATE ${IGN_CXX_11_FEATURES})
 
 if (UNIX)
-  target_link_libraries(ign_msgs_gen pthread)
+  target_link_libraries(gz_msgs_gen pthread)
 endif()
 
-if(INSTALL_IGN_MSGS_GEN_EXECUTABLE)
-  set_target_properties(ign_msgs_gen PROPERTIES VERSION ${PROJECT_VERSION_FULL})
-  install(TARGETS ign_msgs_gen DESTINATION ${IGN_BIN_INSTALL_DIR})
+if(INSTALL_GZ_MSGS_GEN_EXECUTABLE)
+  set_target_properties(gz_msgs_gen PROPERTIES VERSION ${PROJECT_VERSION_FULL})
+  install(TARGETS gz_msgs_gen DESTINATION ${IGN_BIN_INSTALL_DIR})
```

We're changing the name of this executable. Users may be expecting it to be `ign_msgs_gen` when they set `INSTALL_IGN_MSGS_GEN_EXECUTABLE`.

This should work to install the executable in another location:

```suggestion
  install(TARGETS gz_msgs_gen DESTINATION ${IGN_BIN_INSTALL_DIR})
  
  # TODO(chapulina) Deprecated. Remove on v10.
  install(FILES $<TARGET_FILE:gz_msgs_gen> DESTINATION ${IGN_BIN_INSTALL_DIR} RENAME ign_msgs_gen PERMISSIONS OWNER_EXECUTE)
```